submodules/strategyHolidays/strategyHolidays.js

```diff
@@ -413,7 +494,79 @@ define(function(require) {
 			});
 		},
 
-		strategyHolidaysImportDatesFromCsvData: function(parent) {
+		strategyHolidaysIncludeNationHolidaysBindsEvents: function(parent, template, popup) {
+			var self = this;
+
+			template.find('.cancel').on('click', function(event) {
+				popup.dialog('close').remove();
+			});
+
+			template.find('.countries').on('change', function(event) {
+				event.preventDefault();
+
+				var $this = $(this),
+					country = $this.val(),
+					yearSelected = parent.find('#year').val(),
+					holidayDates = new DateHolidays(country),
+					holidays = holidayDates.getHolidays(yearSelected);
+
+				self.strategyHolidaysUpdateNationHolidaysRender(parent, template, holidays);
+			});
+
+			template.on('submit', function(event) {
+				event.preventDefault();
+
+				var $rows = template.find('#include_holidays_table tbody tr .add-holiday:checked'),
+					holidaysListForSelectedYear = self.strategyHolidaysGetHolidaysForCurrentYear(parent, true),
+					holidaysData = _
+						.chain($rows)
+						.map(function(row) {
+							var $tr = $(row).parents('tr');
+
+							return _.pick($tr.data(), [
+								'data',
+								'name'
+							]);
+						})
+						.reject(_.flow(
+							_.partial(_.get, _, 'name'),
+							_.partial(_.includes, holidaysListForSelectedYear)
+						))
+						.value();
+
+				self.strategyHolidaysIncludeHolidaysForCountry(parent, holidaysData);
+				popup.dialog('close').remove();
+			});
+		},
+
+		strategyHolidaysGetHolidaysForCurrentYear: function(parent, isImported) {
+			var self = this,
+				holidaysData = self.appFlags.strategyHolidays.allHolidays,
+				yearSelected = parseInt(parent.find('#year').val()),
+				isCurrentYear = _.flow(
+					_.partial(_.get, _, 'endYear'),
+					_.partial(_.isEqual, _, yearSelected)
+				),
+				isImported = _.partial(_.get, _, 'isImported'),
+				importedHolidaysList = _
+					.chain(holidaysData)
+					.map(function(holiday) {
+						var holidayData = _.get(holiday, 'holidayData');
+
+						return {
+							name: _.get(holidayData, 'name'),
+							endYear: _.get(holidayData, 'endYear', yearSelected),
+							isImported: _.get(holidayData, 'isImported')
+						};
+					})
+					.filter(isCurrentYear)
+					.filter(isImported)
```

Comment:
Instead of filtering twice, we can combine our checks into one with `lodash#overEvery` (and avoid iterating the list twice):

```suggestion
					.filter(_.overEvery(
					    isCurrentYear,
					    isImported
					))
```
This will feed the value to each function and only return `true` if they all return `true`.